release/_bom.sh

```diff
@@ -10,6 +10,8 @@ function download_bnd_files {
 
 	lc_cd "${_BUILD_DIR}/boms"
 
+	rm -f ./*.jar
```

Comment:
@pgydev, can you move all the changes from the file `_bom.sh` in this commit to the previous commit?